### ISSUE TITLE
Embed debug symbols (aka debuginfo) to zulu-openjdk-alpine

### ISSUE THREAD

**Issue by anouarchattouna** [zulu-openjdk/zulu-openjdk#76]
Hello,

Is it possible to embed JDK debug symbols into the zulu-openjdk-alpine ? Otherwise, how could we install those packages ?

Best,

**Comment by kjeet** [comment]
Came here looking for the symbols for the glibc build to use with the async-profiler too. I'm surprised there's been no answer here for years. Even a simple, we do not distribute debug symbols or anything.

**Comment by AntonKozlov** [comment]
Hi, thanks for bumping up. We are bundling symbols in the zulu bundle now

**Comment by rhuitl** [comment]
I came here because I got the message `WARNING: Install JVM debug symbols to improve profile accuracy` with Elastic APM and OpenJDK 17. With Zulu OpenJDK 17 (on Alpine) the message is no longer printed. I think this ticket can be closed.

**Comment by AntonKozlov** [comment]
Thanks for confirming, closing the ticket.

**Comment by rhuitl** [comment]
Updating from 17 to 25 brings up the message
> WARNING: Install JVM debug symbols to improve profile accuracy

again, everything else being equal.

**Comment by azul-os** [comment]
Hi @rhuitl,

We're discussing delivering the needed debug symbols in a seperate package e.g.
```
FROM azul-openjdk-alpine:25

apk add zulu25-jdk-dbg
```

**Comment by rhuitl** [comment]
Makes sense! Elastic APM users who need inferred spans, which brings up this message, can switch to the EDOT agent, see https://discuss.elastic.co/t/elastic-apm-java-agent-on-java-25-with-inferred-spans/383042. I did not encounter this warning message with the EDOT agent.
